### ISSUE TITLE
[Improve]: Allow for remote WordPress plugin installs

### ISSUE THREAD

**Issue by aaronware** [linchpin/actions#24]
Describe the Feature:

Allow for the CI to parse the composer.json file and update and install WordPress plugins to be controlled by the CLI instead of forcing a deployment via composer and rsync.

**Comment by aaronware** [comment]
Worked a bit on the deployment process where we left off on our dev workshop day.  The good news.

1. I was able to get it so setting the flag `skip_composer` will not run `composer update` on the wp-content folder when the project is deployed. And will merely copy whatever plugins are actually part of the repo in the plugins directory
2. I was able to get parse the plugins list from composer.json


**Challenges.** I have not been able to loop through the plugin list and run the SSH command to actually remotely install the plugins. This could be a few things

GitHub Actions use a feature called “matrix” this is basically a loop for all intents and purposes. The matrix I think has to be an array but what we get returned from the json is an object. So this might be one blocker.
Another blocker could be me incorrectly setting the “outputs”. An output is a way to pass data from one job, or step in a job to other parts of the workflow. So I may either be outputting the data incorrectly or inputting the data incorrectly for the next job that tries to install the plugins

I wont be able to toy with it again til next week but if any one has any :linchpinblue: house keeping time. That’s where it’s left off. I was testing on staging for a client but it may make sense to use one of our other non client related workflows unless it makes sense based on what your task is.

All of my work is here https://github.com/linchpin/actions/tree/feature/maybe-skip-composer
